scripts/verify-mo.py

```diff
@@ -20,10 +20,8 @@
 import os
 import shlex
 import subprocess
-from collections.abc import Iterator
 from pathlib import Path
-from types import TracebackType
-from typing import Optional, Set
+from typing import Any, Iterator, Optional, Set
```

Comment:
I wasn't sure how to look this up, so commenting here just to make sure we're not accidentally removing a necessary import: does `Any` supersede `TracebackType`?

Reply:
Yep.  Same deal as freedomofpress/securedrop@6d3f4278e9604091cc0a96756e0836fff872ad3b: We can't get Python < 3.9 (here, 3.8) to accept the syntax in which `TracebackType` is meaningful.